kyuubi-ctl/src/main/scala/org/apache/kyuubi/ctl/opt/AdminCommandLine.scala

```diff
@@ -100,6 +100,6 @@ object AdminCommandLine extends CommonCommandLine {
         arg[String]("<configType>")
           .optional()
           .action((v, c) => c.copy(adminConfigOpts = c.adminConfigOpts.copy(configType = v)))
-          .text("The valid config type can be one of the following: hadoopConf."))
+          .text("The valid config type can be one of the following: hadoopConf, userDefaults."))
```

Comment:
userDefaults => userDefaultsConf

Reply:
OK, let me fix it.
Thus the cli command will become "kyuubi-admin refresh config userDefaultsConf". `conf` occurs twice.